tests/CrawlerRunner/ip_address.py

```diff
@@ -30,9 +29,10 @@ def start_requests(self):
 
     def parse(self, response):
         netloc = urlparse(response.url).netloc
-        self.logger.info("Host: %s" % netloc.split(":")[0])
-        self.logger.info("Type: %s" % type(response.ip_address))
-        self.logger.info("IP address: %s" % response.ip_address)
+        host = netloc.split(":")[0]
+        self.logger.info(f"Host: {host}")
```

Comment:
Just a tip for the future: using single quotes (`':'`) would have worked as well to move `netloc.split(":")[0]` into the f-string.